spec/admin/contribution_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe Contribution do
+  describe 'Contribution' do
+    let(:resource_class) { Contribution }
+    let(:resource) { ActiveAdmin.application.namespaces[:admin].resources[resource_class] }
+
+    it 'Verify resource name' do
```

Comment:
You should use 3rd present person following better specs convention, for example:

```ruby
it 'verifies ...' do; end
```